ocaml/idl/datamodel.ml

```diff
@@ -2318,6 +2318,18 @@ let vm_set_appliance = call
 	~allowed_roles:_R_POOL_OP
 	()
 
+let vm_call_plugin = call
+	~name:"call_plugin"
+	~in_product_since:rel_creedence
```

Comment:
The tag `rel_cream` is not yet defined on this branch, so I use `rel_creedence` here. Would update it once it's ready.